.github/workflows/sitemap-fix.yml

```
@@ -0,0 +1,38 @@
+  workflow_run:
+    workflows: ["Verify sitemap snapshot"]
+    types: [completed]
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  fix:
+    runs-on: ubuntu-latest
+    if: >
```

actionlint (1.7.12)

[error] if: condition "${{ github.event.workflow_run.conclusion == 'failure'\n    && github.event.workflow_run.event == 'pull_request' }}\n" is always evaluated to true because extra characters are around ${{ }}

(if-cond)